apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx

```diff
@@ -23,16 +23,18 @@ const DEFAULT_REVIEW = {
   comments: [
     {
       id: 123,
-      commentText: 'Great work on this!',
-      name: 'Steve',
-      timestampString: '2022-03-31T04:58:42.000Z',
+      comment: 'Great work on this!',
```

Comment:
These comment shape changes are independent of the teacher issue yeah?

Reply:
Oh yes, the proptype shape for the comment was wrong, I was seeing it come up in the console when I was developing, once fixing it, it also needed to be fixed in tests to match the shape.